triton/apps/whisper.rst

```diff
@@ -0,0 +1 @@
+.. include:: ../examples/whisper/whisper.rst
```

Comment:
I would have this in this main page, not included.  Maybe a short full example in the example page and included.  I think it's a bit too long and stand alone to be a full inclusion.